src/main/resources/application.yml

```diff
@@ -175,8 +175,6 @@ azure:
 
 hmpps.sqs:
   queues:
-    migrationcasenotes:
-      propagateTracing: false
```

Comment:
turn on tracing for case notes now that we are not tracing through individual entities